domain/authentication/change.go

```diff
@@ -11,6 +11,6 @@ import (
 //}
 
 func ChangeTokenStorage(tokenStorage token.Token) domain.Token {
-	tokenDomain := domain.Token{ID: tokenStorage.ID, Token: tokenStorage.Token}
+	tokenDomain := domain.Token{ID: tokenStorage.ID, IdAccount: tokenStorage.IdAccount, CreatedAt: tokenStorage.CreatedAt}
 	return tokenDomain
 }
```

Comment:
i believe keeping this conversion is breaking the repository purpose. Wouldn't be better if you keep this conversion, if needed, inside your repository implementation and just return the desired object?